### PR TITLE
Added features updates for ObjectManager

### PR DESCRIPTION
**Pull request by DeyLak**

### CODE REVIEW

**Comment by gribnoysup** [comment]
Hi @DeyLak 

Thank you for your PR! It is good overall, but there are some minor issues I addressed in the review. Would you mind checking them?

File: src/geo-objects/ObjectManager.js [hunk 1]
```diff
@@ -56,6 +57,17 @@ class ObjectManager extends React.Component {
...
 
+    if (!isEqual(features, prevFeatures)) {
+      const prevIds = prevFeatures.map(feature => feature.id)
+      const currentIds = features.map(feature => feature.id)
+      const toBeRemovedIds = difference(prevIds, currentIds)
+      const toBeAddedIds = difference(currentIds, prevIds)
+      const toBeAddedObjects = features.filter(feature => toBeAddedIds.includes(feature.id))
+
+      instance.remove(toBeRemovedIds)
+      instance.add(toBeAddedObjects)
```
Comment (gribnoysup) [inline src/geo-objects/ObjectManager.js]: To clarify, I think that this update could be simplified to 

```js
if (!isEqual(features, prevFeatures)) {
  instance.removeAll()
  instance.add(features)
}

Comment (DeyLak) [inline src/geo-objects/ObjectManager.js]: I tried this on map with ~5k objects, no perf drops were noticed. (Moreover, map itself already wasn't smooth, but that feature replace operation was unnoticable) The propblem with your approach is when you remove all objects and add them again - they all will blink(tested it), and that is, I guess, annoying for user.

Comment (gribnoysup) [inline src/geo-objects/ObjectManager.js]: Okay, that is a valid point! Please move lodash.difference to dev deps and I will merge this

**Pull request merged**